README.md

```diff
@@ -28,12 +28,16 @@ Below are examples of the dice roll syntax.
 
 `!roll 3d6 e6` : Roll three six-sided dice and explode on sixes. Some game systems call this 'open eneded' dice. If the number rolled is greater than or equal to the value given for this option, the die is rolled again and added to the total. If no number is given for this option, it is assumed to be the same as the number of sides on the die. Thus, '3d6 e' is the same as '3d6 e6'.
```

Comment:
Just noticed this as well. Should the verbiage be updated here that explodes will only happen once now with this command?